doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -424,6 +424,10 @@ Multiprotocol Service Layer libraries
 Libraries for networking
 ------------------------
 
+* :ref:`lib_mqtt_helper` library:
```

Comment:
Remove this changelog entry and ask @divipillai to add it to her https://github.com/nrfconnect/sdk-nrf/pull/18192